### PR TITLE
import a small datepicker and use it in contact forms

### PR DESCRIPTION
**Pull request by rymaju**
Changes:

- Used this npm library for the date picker
- range from 100 years ago to current year
- works on mobile
- Need to check if it works on safari

image
image
image

### CODE REVIEW

File: src/components/Forms/MainContactForm.vue [hunk 1]
```diff
@@ -221,7 +226,7 @@ export default {
 };
 </script>
 
-<style lang="less" scoped>
+<style lang="less">
```
Comment (rymaju) [inline src/components/Forms/MainContactForm.vue:229]: Needed to do this for selectors to work properly, sorry 😬

File: src/components/Forms/AdditionalContactForm.vue [hunk 2]
```diff
@@ -250,7 +255,13 @@ export default {
     font-size: 1rem;
     margin-left: 6px;
   }
-
+  .input-datepicker {
+    font-size: 10px;
+  }
+  .input-datepicker::v-deep input {
```
Comment (jackblanc) [inline src/components/Forms/AdditionalContactForm.vue:261]: ❓ What's v-deep?

**Review by jackblanc — APPROVED** [review]
verified on safari 🚢

File: src/components/Forms/AdditionalContactForm.vue (hunk 2 above)
Comment (rymaju) [reply src/components/Forms/AdditionalContactForm.vue:261]: Long story...

Basically "scoped" in vue stops us from using the > selector. What I wanted to write was something along the lines of:

```css
.input-datepicker > input {
    ...
}
```
But it didnt like that, turns out you need to use the v-deep selector to get around that.

Vue bad. 😠

**Pull request merged**